src/main/scala/io/iohk/ethereum/faucet/jsonrpc/FaucetBuilder.scala

```diff
@@ -84,6 +87,21 @@ trait FaucetJsonRpcHttpServerBuilder {
   )
 }
 
+trait ShutdownHookBuilder {
+  self: ActorSystemBuilder with Logger =>
+
+  def shutdown(): Unit = {
+    import scala.concurrent.duration._
+    Await.ready(
+      system.terminate.map(
+        _ ->
+          log.info("actor system finished")
+      )(system.dispatcher),
+      1.seconds
```

Comment:
why not use the pre-existing mantis config value?
```  
# timeout for shutting down the ActorSystem
  shutdown-timeout = "15.seconds"
```

Reply:
i mean as in `src/main/scala/io/iohk/ethereum/nodebuilder/NodeBuilder.scala`:
```
trait ShutdownHookBuilder {
  self: Logger =>
  ...
  lazy val shutdownTimeoutDuration: Duration = Config.shutdownTimeout
...
}
```

Reply:
Done